server/src/termdb.server.init.js

```diff
@@ -68,10 +69,15 @@ export function server_init_db_queries(ds) {
 		'buildDate'
 	]
 
-	if (tables.has('sample_ancestry')) {
-		const num_rows = cn.prepare('SELECT count(*) as num_rows FROM sample_ancestry').get().num_rows
-		if (num_rows > 0) ds.cohort.termdb.hasAncestry = true
+	ds.types = new Map()
```

Comment:
to be consistent with nearby code, should attach `types` not directly on `ds`, but maybe `ds.cohort.termdb`?